requirements.txt

```diff
@@ -12,5 +12,4 @@ pypng>=0.0.20
 pystache>=0.5.4,<0.6.0
 requests>=2.25.0,<3.0.0
 rsa>=4.7,<5.0.0
-xgboost>1.1.0
-sklearn
\ No newline at end of file
+sklearn
```

Comment:
might also want to move this to `xgboost_install_requires`

Reply:
also I think it's `scikit-learn` not `sklearn`

Reply:
hog/runs for now and then a merge into hog/main once it's in good shape?

Reply:
sounds good 👍🏼